gtsfm/utils/images.py

```diff
@@ -41,7 +41,7 @@ def rgb_to_gray_cv(image: Image) -> Image:
     return Image(output_array, image.exif_data)
 
 
-def vstack_images(image_i1: Image, image_i2: Image) -> Image:
+def vstack_image_pair(image_i1: Image, image_i2: Image) -> Image:
```

Comment:
can we just use `vstack_image_list`?

Reply:
Sure. The only difference though is that for the image pair, you had functionality to resize to the same width; for the stacking image list I assumed they were all of the same width. But do you think it is cleaner  to consolidate them?

Reply:
Oh I got it. Yeah lets keep it separate for now.